docs/meshstack.configuration.md

```diff
@@ -191,15 +191,19 @@ Operators can configure the mailbox this feedback is sent to via `meshfed.web`:
 
 When you add users to your [meshCustomers](./meshcloud.customer.md) we currently support live typeahead for users stored in an Azure AAD Identity Provider and Google Cloud Directory (GCD). This makes it easier for people to invite additional users without remembering their full contact details.
 
-#### Configure for Azure AAD
+#### Configure Azure AAD
 
-In order to use this functionality, you must create a new principal (described in **Replicator** &rarr; **AAD Level Permissions** step 1 and 2) and assign the following required permissions as an **application permission**:
+In order to use this functionality, create a new principal (described in **Replicator** &rarr; **AAD Level Permissions** step 1 and 2) and assign the following required permissions:
```

Comment:
d: Shouldnt this whole section rather moved to https://docs.dev.meshcloud.io/docs/meshstack.configuration.html#user-identity-lookup? To avoid this double documentation. In the end this describes on how to setup IDP lookup which is only used in meshfed-api so I voted to move the docs there. Or we should remove it form there. You can decide. :D

Reply:
yes you are right we do not need the duplication